daemon/algod/api/server/v2/delta.go

```diff
@@ -70,6 +71,57 @@ func convertAssetResourceRecordToGenerated(asset ledgercore.AssetResourceRecord)
 	}
 }
 
+// accountDataToAccount converts a ledgercore.AccountData to model.Account
+func accountDataToAccount(addr string, amntWithoutPendingRewards uint64, rnd uint64,
+	consensus *config.ConsensusParams, ad ledgercore.AccountData) (model.Account, error) {
+	pendingRewards, overflowed := basics.OSubA(ad.MicroAlgos, basics.MicroAlgos{Raw: amntWithoutPendingRewards})
+	if overflowed {
+		return model.Account{}, errors.New("overflow on pending reward calculation")
+	}
+	var apiParticipation *model.AccountParticipation
+	if ad.VoteID != (crypto.OneTimeSignatureVerifier{}) {
+		apiParticipation = &model.AccountParticipation{
+			VoteParticipationKey:      ad.VoteID[:],
+			SelectionParticipationKey: ad.SelectionID[:],
+			VoteFirstValid:            uint64(ad.VoteFirstValid),
+			VoteLastValid:             uint64(ad.VoteLastValid),
+			VoteKeyDilution:           ad.VoteKeyDilution,
+		}
+		if !ad.StateProofID.IsEmpty() {
+			tmp := ad.StateProofID[:]
+			apiParticipation.StateProofKey = &tmp
+		}
+	}
+	var authAddr *string = nil
+	if !ad.AuthAddr.IsZero() {
+		authAddr = strOrNil(ad.AuthAddr.String())
+	}
+	return model.Account{
```

Comment:
Might be worth putting a comment cross linking the `model.Account` init in `AccountDataToAccount` so that the next person in this code knows to update both.

Reply:
Ended up moving it to `account.go` since it sort of just belongs there.